puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_develop.sh

```diff
@@ -44,3 +43,10 @@ bundle exec rake db:drop db:create db:migrate --trace
 tasks="pkg:generate_source jenkins:unit"
 [ ${database} = postgresql ] && tasks="$tasks jenkins:integration"
 bundle exec rake $tasks TESTOPTS="-v" --trace
+
+# Test asset precompile
+if [ ${database} = postgresql -a -e "$APP_ROOT/package.json" ]; then
+  bundle exec rake db:drop db:create db:migrate RAILS_ENV=production
```

Comment:
In the past we've seen you can't do this in 1 line. I think the `db:migrate` needs to be a separate rake invocation due to caching.

Reply:
We may need to explicitly drop the production database here https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/gemset_cleanup.sh#L20

Reply:
@ehelms so I should add the db:drop with RAILS_ENV=production there, correct?

Reply:
I'd prefer `gemset_cleanup` not to do any rails tasks. Currently the kafo job is already broken because of it (needs `PUPPET_VERSION` to be set for bundle exec to work and the `|| true` doesn't appear to work). Can we introduce a `rails_cleanup` instead and use that where needed?